agent/agent.go

```diff
@@ -3840,6 +3845,21 @@ func (a *Agent) getPersistedTokens() (*persistedTokens, error) {
 	return persistedTokens, nil
 }
 
+// CheckSecurity Performs security checks in Consul Configuration
+// It might return an error if configuiration is considered too dangerous
+func (a *Agent) CheckSecurity(conf *config.RuntimeConfig) error {
+	if conf.EnableRemoteScriptChecks {
+		if conf.ACLsEnabled {
```

Comment:
```suggestion
		if !conf.ACLsEnabled {
```

Reply:
Good catch, I changed test, fixed in force-pushed commit